bitnami/etcd/templates/cronjob-defrag.yaml

```diff
@@ -115,9 +115,6 @@ spec:
               {{- else if ne .Values.defrag.cronjob.resourcesPreset "none" }}
               resources: {{- include "common.resources.preset" (dict "type" .Values.defrag.cronjob.resourcesPreset) | nindent 16 }}
               {{- end }}
-              {{- if .Values.defrag.cronjob.containerSecurityContext.enabled }}
-              securityContext: {{- include "common.compatibility.renderSecurityContext" (dict "secContext" .Values.defrag.cronjob.containerSecurityContext "context" $) | nindent 16 }}
-              {{- end }}
```

Comment:
I would say that this is the containerSecurityContext that should stay. 

However, that would cause more ambiguity with the snapshooter job, which uses `.Values.containerSecurityContext` and `.Values.podSecurityContext`.

Could you please perform the following changes?
- Update the snapshooter cronjob to use `disasterRecovery.cronjob.containerSecurityContext` and `defrag.cronjob.podSecurityContext`?
- To declare the values in the `values.yaml`, you can copy them from either `.containerSecurityContext/podSecurityContext` or `defrag.cronjob.containerSecurityContext/podSecurityContext`.
- Bump a minor version instead of a patch.

If I'm asking too much, please let me know and I will implement them in a separate PR.

Thank you very much